src/node/transaction.cpp

```diff
@@ -13,6 +13,18 @@
 
 #include <future>
 
+TransactionError HandleATMPError(TxValidationState state, std::string& err_string) {
```

Comment:
Minor style suggestions (no need to change these unless you're retouching the branch anyway):

- Make the `TxValidationState state` argument a const reference to avoid a copy on the stack.
- This function is only used internally in this translation unit. Make it `static` or put it in an unnamed namespace to avoid external linkage.

Reply:
done, thanks!